site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,552 @@
+An introduction to Sphinx and Read the Docs for Technical Writers
+=================================================================
+
+By: Eric Holscher
+
+Bio: Eric Holscher is the co-founder of Read the Docs, and the Write the Docs conference.
+Along with community organizing and open source work,
+he does consulting around software documentation and speaks at a number of industry events each year.
+When he isn't expressing his views on software docs, he's getting views from the top of mountains.
+
+Treating documentation as code is becoming a major theme in the software industry.
+This is coming from both sides,
+with developers starting treat documentation as a priority alongside tests and code,
+and writers seeing a lot of value in integrating more into the development process.
+This marrying of cultures isn't simple,
+but having the proper tools for the job makes both sides happy with the results that get produced.
+
+A lot of developer tools don't work well for writers.
+Sphinx and Read the Docs are unique in this ecosystem,
+in that they have powerful tools that both writers and developers want,
+in one convenient package.
+
+Overview of the Ecosystem
+-------------------------
+
+Read the Docs is the largest open source documentation hosting site in the world.
+Open Source in this context means both that the code is open source,
+and the documentation hosted is for open source software.
+It is a developer focused platform,
+but it has most of the features that technical writers have come to expect in a tool as well.
+This blending of worlds makes it the best suited platform for teams that want both writers and developers contributing to product and API documentation.
+
+Read the Docs is best thought of as a continuous documentation platform for Sphinx.
+Continuous documentation is analogous to continuous integration of code.
+In practice it means that your documentation is built,
+tested, 
+and updated on every commit.
```

Comment:
I think for the layman, this paragraph can be clarified a little.

* Maybe you're familiar with continuous integration from the testing world
* If not, here is what it is
* Read the Docs is continuous integration for documentation -- think of it as continuous documentation
* This is what continuous documentation essentially does